integrations/aws-iam/syncs/users.ts

```diff
@@ -0,0 +1,113 @@
+import type { NangoSync, ProxyConfiguration, User } from '../../models';
+import { getAWSAuthHeader } from '../helper/utils.js';
+import type { AWSIAMRequestParams, AWSIAMUser, TagMember, ListUsersResponse, ListUserTagsResponse } from '../types';
+
+export default async function fetchData(nango: NangoSync) {
+    const method = 'GET';
+    const service = 'iam';
+    const path = '/';
+    const params = {
+        Action: 'ListUsers',
+        Version: '2010-05-08'
+    };
+
+    // Set AWS IAM parameters
+    const requestParams: AWSIAMRequestParams = {
+        method,
+        service,
+        path,
+        params
+    };
+
+    // https://docs.aws.amazon.com/IAM/latest/APIReference/API_ListUsers.html
+    for await (const awsUsers of paginate<AWSIAMUser>(nango, requestParams)) {
+        const users: User[] = [];
+
+        for (const user of awsUsers) {
+            const tags = await fetchUserTags(nango, user.UserName);
```

Comment:
in AWS-IAM, they usually dont have these standard fields, as a walkaround, they recommend attaching tags to a new user, which can then be used during `users` sync.